sds_data_manager/utils/stackbuilder.py

```diff
@@ -92,62 +93,49 @@ def build_sds(scope: App, env: Environment, account_config: dict):
         env=env,
         vpc=networking.vpc,
         rds_security_group=networking.rds_security_group,
-        engine_version=rds.PostgresEngineVersion.VER_14_2,
+        engine_version=rds.PostgresEngineVersion.VER_15,
```

Comment:
If you rebase with latest change, it will look like this. The latest version CDK supports is version 15.3. Sean and we both tried and that seems to the latest at the moment.
```suggestion
        engine_version=rds.PostgresEngineVersion.VER_15_3,
```